src/services/profile/updateUser.ts

```diff
@@ -0,0 +1,44 @@
+import { Request, Response } from "express";
+import User from "../../models/userModel";
+const { StatusCodes } = require('http-status-codes');
+
+import { error } from "console";
+import UserType from "../../interfaces/userType";
+
+const TOKEN_SECRET = String(process.env.TOKEN_SECRET);
+
+
+const updateUser = async (req:Request, res:Response) => {
```

Comment:
You're suppose to return updated user from here not response. Controller is responsible for response